### ISSUE TITLE
Internal Server Error when uploading file with non-ascii filename in a form

### ISSUE THREAD

**Issue by widder** [stephenmcd/mezzanine#1862]
When uploading a file like for example `https://ihl-databases.icrc.org/applic/ihl/ihl-nat.nsf/0/abf9bb704d4adec5412566e200524e21/$FILE/B%C3%BCrgerliches%20Gesetzbuch.pdf`

this error occurs:

```
Internal Server Error: /de/testform/

UnicodeEncodeError at /de/testform/
'ascii' codec can't encode character '\u0308' in position 71: ordinal not in range(128)
```

**Comment by stephenmcd** [comment]
You've probably not correctly configured UTF support in your environment (web server, db, etc). This has been discussed exhaustively on the mailing list.

**Comment by widder** [comment]
I could not found a solution on the mailing list concerning this issue on non-ascii filenames.

I found this thread https://groups.google.com/forum/#!searchin/mezzanine-users/filename|sort:date/mezzanine-users/7mANDYTdgv4/TJznu_OLCgAJ
with no answer.

I found this PR https://github.com/stephenmcd/mezzanine/pull/1609/files but it is only about uploading ZIPs. Instead, I have this problem with a PDF file.

Locale, Webserver, Operating System and Postgres are all set to UTF-8.
